src/targets/gpu/CMakeLists.txt

```diff
@@ -33,11 +33,14 @@ if(NOT TARGET MIOpen)
     message(SEND_ERROR "Cant find miopen")
 endif()
 
+set(MIGRAPHX_USE_HIPRTC OFF CACHE BOOL "Use hipRTC APIs")
+
 include(Embed)
 file(GLOB KERNEL_FILES ${CONFIGURE_DEPENDS}
     ${CMAKE_CURRENT_SOURCE_DIR}/kernels/include/migraphx/kernels/*.hpp)
 message(STATUS "KERNEL_FILES: ${KERNEL_FILES}")
 add_embed_library(migraphx_kernels ${KERNEL_FILES})
+target_compile_definitions(migraphx_kernels PRIVATE -DMIGRAPHX_USE_HIPRTC)
```

Comment:
This flag shouldnt be added to the `migraphx_kernels`.

Reply:
Done